src/components/internal/CompoundField.tsx

```diff
@@ -0,0 +1,66 @@
+/** Internal component to help create compound fields */
```

Comment:
Looks like this snuck up to the top (happens with auto-add import sometimes)

Reply:
Thanks